### PR TITLE
W-14066738-midlevelpermissionsbeforeyoubegin-se

### PR DESCRIPTION
**Pull request by sebermuley**
Added mid-level permissions to the permissions tables in the Before You Begin section on every page

### CODE REVIEW

File: rpa-manager/modules/ROOT/pages/alerting-rule.adoc [hunk 1]
```diff
@@ -4,16 +4,18 @@ Specify the incidents that trigger an alert. Each activated rule be checked whil
...
+* Ask an organization administrator in Access Management to assign you the required permissions:
 +
-[cols="1,2"]
+[cols="1,1,1"]
 |===
-|*Action* |*Required Privileges*
+|*Action* |*RPA Permission* | *Deprecated RPA Permission*
 
 |Open the *Alerting* module
+|RPA Operations Viewer, RPA Operations Manager, RPA Administrator
```
Comment (Crispy-Salesforce) [inline rpa-manager/modules/ROOT/pages/alerting-rule.adoc:14]: Would it be better to say?:
"RPA Operations Viewer, RPA Operations Manager, and RPA Administrator"
If so, make the change globally.

File: rpa-manager/modules/ROOT/pages/botmanagement-know.adoc [hunk 2]
```diff
@@ -88,14 +88,15 @@ The bot has been configured for a desktop session. It cannot use secure sessions
...
 +
-[cols="1,2"]
+[cols="1,1,1"]
 |===
-|*Action* |*Required Privileges*
+|*Action* |*RPA Permission* | *Deprecated RPA Permission*
 
 |Open the *Bot Management* module and view the bots
-|Bot Management Open
+|RPA Bots Manager, RPA Operations Viewer, RPA Operations Manager, RPA Administrator
```
Comment (Crispy-Salesforce) [inline rpa-manager/modules/ROOT/pages/botmanagement-know.adoc:98]: Would it be better to say?:
"RPA Bots Manager, RPA Operations Viewer, RPA Operations Manager, and RPA Administrator"
If so, make the change globally.

File: rpa-manager/modules/ROOT/pages/myrpa-start.adoc [hunk 3]
```diff
@@ -10,13 +10,14 @@ You can start the automation of a process as a project manager from your backlog
...
+* Ask an organization administrator in Access Management to assign you the required permissions:
 +
-[cols="1,2"]
+[cols="1,1,1"]
 |===
-|*Action* |*Required Privileges*
+|*Action* |*RPA Permission* | *Deprecated RPA Permission*
 
 |Start automation of a process.
+|RPA Automations Designer, RPA Automations Contributor, RPA Automations Manager, RPA Administrator
```
Comment (Crispy-Salesforce) [inline rpa-manager/modules/ROOT/pages/myrpa-start.adoc:20]: Would it be better to say?:
"RPA Automations Designer, RPA Automations Contributor, RPA Automations Manager, and RPA Administrator"
If so, make the change globally.

**Review by Crispy-Salesforce — APPROVED** [review]

File: rpa-manager/modules/ROOT/pages/alerting-rule.adoc (hunk 1 above)
Comment (sebermuley) [reply rpa-manager/modules/ROOT/pages/alerting-rule.adoc:14]: asked Diane and decided against

**Pull request merged**